src/main/resources/application.xml

```diff
@@ -1,55 +0,0 @@
-quarkus:
-  http:
-    port: 8080
-    read-timeout: 30m
-    limits:
-      max-body-size: 500M
-
-  package:
-    uber-jar: true
-
-  # Logging (disable console on prod)
-  log:
-    level: INFO
-    min-level: INFO
-    category:
-      "org.jboss":
-        level: WARN
-      "io.quarkus":
-        level: WARN
-      "io.smallrye":
-        level: WARN
-      "org.eclipse":
-        level: WARN
-      "io.netty":
-        level: WARN
-      "org.infinispan":
-        level: WARN
-      "io.agroal":
-        level: WARN
-      "org.commonjava.indy.service":
-        level: INFO
-    console:
-      enable: false
-    file:
-      level: INFO
-      enable: true
-      path: "log/indy-archive-service.log"
-      format: "%d{HH:mm:ss} %-5p [%c{2.}] (%t) %s%e%n"
-      rotation:
-        max-backup-index: 5
-        max-file-size: 10M
-
-  swagger-ui:
-    always-include: true
```

Comment:
I think this file should not be removed but just to rename to yml with your items merging